app/views/admin/users/_reports.html.erb

```diff
@@ -0,0 +1,21 @@
+<div class="row my-3">
+  <div class="col-12">
+    <h2 class="d-inline">Recent Reports</h2>
+    <button type="button" data-toggle="collapse" data-target="#reactions-row" class="btn btn-secondary float-right">Toggle</button>
+  </div>
+  <div class="col-12 collapse" id="reactions-row">
+    <div class="list-group-flush">
+    <% unless @related_reports.empty? %>
+      <% @related_reports.each do |report| %>
+        <a href="<%= report.reported_url %>" class="list-group-item list-group-item-action action px-5 d-flex justify-content-between">
```

Comment:
Hey! So I checked the format of `reported_url` in production, and the URLs are absolute (eg `https://dev.to/weber360/comment/cble`).

However, I notice in this PR that `<%= report.reported_url %>` is generating a relative URL.

Assume `report.reported_url` is `example.com`. The `href` above is generating `localhost:3000/admin/users/:id/example.com` instead of `example.com`, and of course, the report's link is blowing up when clicked.

<img width="1426" alt="Screen Shot 2020-08-31 at 11 54 45 AM" src="https://user-images.githubusercontent.com/32520970/91740073-c6b03780-eb80-11ea-9706-51ae93a2b158.png">

I'm still exploring why, but wanted to give this heads-up. 👍

Reply:
Yeah, I'm unsure of what to use here or if I should remove the link entirely -- I was waiting for everyone's opinions when they review this, so sounds good! Thanks, @msarit 👍

Reply:
I've since changed this and now have each individual report linking to the `Feedback Messages` page, `/admin/feedback_messages`, which should give Admins the context they need per our discussion during our team sync today! Let me know what you think @msarit. 😄